src/adapter/templates/getStringyProps.ts

```diff
@@ -24,7 +29,12 @@ export const getStringyProps = remoteFunction(function (this: unknown, maxLength
 });
 
 export const getToStringIfCustom = remoteFunction(function (this: unknown, maxLength: number) {
-  if (typeof this === 'object' && this && !String(this.toString).includes('[native code]')) {
+  if (
+    typeof this === 'object' &&
+    this &&
+    !String(this.toString).includes('[native code]') &&
+    !String(this).includes('[object ')
```

Comment:
In some cases, calling `String(this)` may be expensive. We should call it when the first three conditions pass and nest an `if` to return.

```
const str = String(this)
if (!str.startsWith('[object ')) {
  return str.slice(0, maxLength)
}
```